website/source/resources.html.erb

```diff
@@ -0,0 +1,94 @@
+  <li><a href="https://www.hashicorp.com/resources/pagerduty-nomad-journey"><strong>PagerDuty's</strong> Nomadic Journey</a></li>
+  <li><a href="https://www.hashicorp.com/resources/operating-jobs-scale-nomad">Operating jobs at scale with Nomad</a></li>
+  <li><a href="https://www.hashicorp.com/resources/backend-batch-processing-nomad">Backend batch processing with Nomad</a></li>
+</ul>
+
+<h2>Blog Posts</h2>
+
+<ul>
+  <li><strong><a href="https://medium.com/@gabrielteles/migrating-from-aws-to-aws-e2bb1bfaa0e0">Migrating from AWS to AWS</a> - Gabriel Teles</strong></li>
+  <li><strong><a href="https://stackshare.io/circleci/how-circleci-processes-4-5-million-builds-per-month">How CircleCI Processes 4.5 Million Builds Per Monthd</a> - Rob Zuber</strong></li>
+  <li><strong><a href="http://timperrett.com/2017/05/13/nomad-with-envoy-and-consul">Envoy with Nomad and Consul</a> - Tim Perrett</strong></li>
+  <li><strong><a href="https://www.hashicorp.com/blog/continuous-deployment-with-nomad-and-terraform">Continuous Deployment with Nomad and Terraform</a> - Nic Jackson</strong></li>
+  <li><strong><a href="https://www.hashicorp.com/blog/running-apache-spark-on-nomad">Running Apache Spark on HashiCorp Nomad</a> - Rob Genova</strong></li>
+  <li><strong><a href="https://www.hashicorp.com/blog/auto-bootstrapping-a-nomad-cluster">Auto-bootstrapping a Nomad Cluster</a> - Nic Jackson</strong></li>
+  <li><strong><a href="https://www.hashicorp.com/blog/replacing-queues-with-nomad-dispatch">Replacing Queues with Nomad Dispatch</a> - Armon Dadgar</strong></li>
+  <li><strong><a href="https://sanderknape.com/2016/08/nomad-consul-multi-datacenter-container-orchestration/">Multi-datacenter Container Orchestration with Nomad and Consul</a> - Sander Knape</strong></li>
+</ul>
+
+<h2>Tools for Provisioning and Experimentation</h2>
+
+<ul>
+  <li><strong><a href="https://registry.terraform.io/search?q=nomad">Terraform Module Registry</a> - Official Nomad modules for AWS, Azure and Google Cloud</strong></li>
+  <li><strong><a href="https://aws.amazon.com/quickstart/architecture/nomad/">HashiCorp Nomad on AWS</a> - AWS Quickstart Guide for Nomad</strong></li>
+  <li><strong><a href="https://github.com/hashicorp/nomad/tree/master/terraform">Nomad Sandbox Environment for AWS and Azure</a> - Easily Provision a Combined Nomad, Consul and Vault Cluster on AWS or Azure</strong></li>
+  <li><strong><a href="https://www.katacoda.com/hashicorp/scenarios/nomad-introduction">Introduction to Nomad (Katacoda)</a> - Interactive Introduction to Nomad</strong></li>
+  <li><strong><a href="https://katacoda.com/hashicorp/scenarios/playground">Nomad Playground (Katacoda)</a> - Online Nomad Playground Environment</strong></li>
+</ul>
+
+<h2>Integrations</h2>
+
+<ul>
+  <li><strong><a href="https://verizon.github.io/nelson/">Verizon Nelson</a> - Automated, Multi-region Container Deployment with HashiCorp Nomad</strong></li>
```

Comment:
DataDog would be a good one to add

Reply:
It doesn't appear to be live on their site yet.